### ISSUE TITLE
Flow

### ISSUE THREAD

**Issue by boshinski** [Templarian/MaterialDesign#2812]
Need an icon representing flow. Was thinking about three wavy lines (like the waves icon) but with arrow heads on the ends. I could see four of these. One for each direction. Flowing right, Flowing left, flowing up (like evaporation) or flowing down

**Comment by Croutonix** [comment]
![image](https://user-images.githubusercontent.com/18645219/37561541-0dea04f0-2a27-11e8-8213-22bbf3d50619.png)
There are three more for left, up and down: [download link](https://github.com/Templarian/MaterialDesign/files/1822255/icons.zip)

**Comment by CoDEmanX** [comment]
The arrow heads are hard to read. I tried a couple of things to make them more legible but to no avail 😕

**Comment by goyney** [comment]
I don't think this icon really works in its current form. Moving to the Backlog for now.

**Comment by goyney** [comment]
Here's my thoughts on this one... Ditch the whole wave arrows completely because they tend to look terrible at native resolution and go this route:
![image](https://user-images.githubusercontent.com/1015040/131731597-9cc3c604-1928-4b69-8ef7-66783634d317.png)![image](https://user-images.githubusercontent.com/1015040/131731623-ed27dfea-db1b-4ee0-bd10-def7734cb3bb.png)
![image](https://user-images.githubusercontent.com/1015040/131731665-8757a3df-2c5c-47a2-9869-03d771a17324.png)

These can be used to represent flow forward and backwards, tides in and out, and lastly evaporation.
[Archive.zip](https://github.com/Templarian/MaterialDesign/files/7093798/Archive.zip)
